webapp/src/main/java/org/opentestsystem/rdw/admin/repository/GroupImportRepository.java

```diff
@@ -0,0 +1,24 @@
+package org.opentestsystem.rdw.admin.repository;
+
+/**
+ * This repository is responsible for working with imports.
+ */
+public interface GroupImportRepository {
+
+    /**
+     * Create a Import record to represent a warehouse
+     * to data-mart Group migration.
+     *
```

Comment:
Perhaps additional verbiage that says something like, "Create an Import record to associate with new group data; the record is created with status ACCEPTED"